apparmor.d/profiles-m-r/murmurd

```diff
@@ -0,0 +1,53 @@
+# apparmor.d - Full set of apparmor profiles
+# SPDX-License-Identifier: GPL-2.0-only
+
+include <tunables/global>
+
+@{exec_path} = /{,usr/}{,s}bin/murmurd
+profile murmurd @{exec_path} {
+  include <abstractions/base>
+  include <abstractions/nameservice-strict>
+  include <abstractions/openssl>
+  include <abstractions/dbus-strict>
+  include <abstractions/ssl_certs>
+
+  capability chown,
+  capability setgid,
+  capability setuid,
+  capability dac_override,
+  capability sys_resource,
+
+  network inet stream,
+  network inet6 stream,
+  network inet dgram,
+  network inet6 dgram,
+
+  @{exec_path} mr,
+
+  /{,usr/}bin/lsb_release Px -> lsb_release,
+
+  dbus send bus=system path=/
```

Comment:
Try to put the `dbus` rules before the `@{exec_path} mr,`

Reply:
Sorry, got confused with recent style change.